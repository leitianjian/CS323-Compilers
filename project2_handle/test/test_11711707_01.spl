struct ST{
    int arr[4][2], arr1[4], val1;
    struct Inner{
        float f;
        int arr1[4];
    } inner1[2], inner2;
} s;

int main(){
    s.arr2[3] = 2;
    s.arr[2] = s.arr1;
    s.arr1[2][1] = 1;
    s.arr1[1] = 0;
    s.val1 = 3.0;
    s.arr1 = s.inner2.arr1;
    s.arr1 = s.inner1[1].arr1;
    s.val1 = s.inner2.f;
    return 1;
}