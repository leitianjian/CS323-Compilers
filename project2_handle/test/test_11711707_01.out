Error type 13 at Line 10: accessing an undefined structure member
Error type 10 at Line 10: applying indexing operator ([...]) on non-array type variables
Error type 5 at Line 10: unmatching type on both sides of assignment
Error type 10 at Line 12: applying indexing operator ([...]) on non-array type variables
Error type 5 at Line 12: unmatching type on both sides of assignment
Error type 5 at Line 14: unmatching type on both sides of assignment
Error type 5 at Line 17: unmatching type on both sides of assignment
